ruspy/estimation/estimation_cost_parameters.py

```diff
@@ -90,9 +90,8 @@ def loglike_opt_rule(
     :return: The negative loglikelihood value for minimizing the objective function.
     """
     costs = cost_func(num_states, maint_func, params)
-    # ev = calc_fixp(num_states, trans_mat, costs, beta, max_it=max_it)
-    # p_choice = choice_prob(ev, costs, beta)
-    p_choice = converge_choice(num_states, trans_mat, costs, beta, max_it=max_it)
+    ev = calc_fixp(num_states, trans_mat, costs, beta, max_it=max_it)
```

Comment:
Please change order of `max_it` and `threshold` in the `calc_fixp` signature. Then you can just pass in `max_it`and it will be matched properly due to its position.

Reply:
Ok